earth_enterprise/src/fusion/autoingest/StorageManager.h

```diff
@@ -38,9 +43,55 @@ class StorageManaged {
     StorageManaged() : timestamp(0), filesize(0) {}
 };
 
+template<class AssetType, class AssetStorageType> class AssetHandleInterface;
+
+template<class AssetType, class AssetStorageType>
+class AssetSerializerLocalXML
+{
+  public:
+    AssetSerializerLocalXML() {}
+    std::shared_ptr<AssetType> Load(const std::string &boundref)
+    {
+      return AssetType::Load(boundref);
+    }
+
+    bool Save(std::shared_ptr<AssetType> asset, std::string filename){
+      extern void ToElement(DOMElement *elem, const AssetStorageType &self);
+
+      std::unique_ptr<GEDocument> doc = CreateEmptyDocument(asset->GetName());
+      if (!doc) {
+          notify(NFY_WARN,
+                "Unable to create empty document: ${name}AssetVersion");
```

Comment:
This warning should call asset->getName() instead of printing `${name}AssetVersion`.

Reply:
Good catch.